jupyterhub_moss/form/option_form.js

```diff
@@ -13,6 +13,14 @@ function setVisible(element, visible) {
   }
 }
 
+function setReadOnly(element, readonly) {
```

Comment:
It looks `readonly` behaves as `hidden`.

Reply:
What you do mean by that ?

Reply:
It's not a boolean value you can set, you need to set/remove the attribute